README.md

```diff
@@ -153,23 +153,24 @@ dependencies {
 
 ##### Set the authority
 
-To set the authority you need to override the string resource of the library with `resValue` in your `build.gradle`
+To set the unique authority you need to set applicationId in your `build.gradle`
 ```java
 android {
     ...
     defaultConfig {
-        applicationId "your.app.id" // this is your unique applicationId
-
-        resValue "string", "tray__authority", "${applicationId}.tray" // add this to set a unique tray authority based on your applicationId
+        applicationId "your.app.id" // this is your unique applicationId as authority prefix
```

Comment:
we should remove this `applicationId` completely except for the override part below